questions/accessibility-aria.md

```diff
@@ -13,16 +13,17 @@ An example using ARIA:
   aria-owns="ex1-grid"
   aria-haspopup="grid"
   id="ex1-combobox">
+  <!-- more code here-->
```

Comment:
Wasn't there an elipses here? Is that why it's conflicting?

Reply:
I was torn about clarity. @skatcat31 does the updated description and bullet points make more sense to you?

Reply:
No I mean... I was seeing a conflict and no ellipses. I was wondering if that was why the PR said there was a conflict

Reply:
You were right though. I resolved that conflict. I changed to a comment because I was torn about the clarity of how to present the idea there was more code that would go between the opening and closing `div`.